participants/filecoin.net-disk.md

```diff
@@ -0,0 +1,56 @@
+# <Project Name>
+Filecoin.Net Disk
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+My project name is Filecoin.Net Disk
+My slack username is cannon, slack emil is cannon3758@outlook.com
+
+## (1) Brief Project Description
+This project is provide data store for users, which is use web UI interface to operation.
+And then we use lotus for data storage and retrieve deal from filecoin blockchain 
+
+
+## (2) Link to Project UI
+http://183.60.252.190:3334
+
+## (3) What does your application/UI do?
+Users can storage or retrieve data from filecoin network by this application 
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+My data is  sorts of video , we have the rights to the data for my plan. We have the data license available online. The data does'nt contain any  sensitive or illegal information 
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We have probably 100T data planning  store to the Filecoin Network
+
+## (7) How are you structuring the data?
+Our data is manly for video format such as mp4, avi, and so on 
```

Comment:
I see a lot of QLV files, which are not the most common video formats. Is there some way you can convert to MP4/AVI/etc before uploading?

Reply:
The qlv file  just a small part，  most of them are the mkv format， but maybe upload after a while

Reply:
Why is my proposal still not passed？could  you give me some advice？

Reply:
It looks like you still need to add your Slack username handle, see: https://github.com/filecoin-project/slingshot/pull/13#issuecomment-698595896

Reply:
My slack username is cannon, slack emil is cannon3758@outlook.com

Reply:
I have add to my pr participants/filecoin.net-disk.md file, please  check it

Reply:
I have added it in participants/filecoin.net-disk.md file  long ago,  
My slack username is cannon, slack emil is cannon3758@outlook.com

Reply:
Thank you for your review， but why I could not found logo and rank with my wallet address  in the leadboard ?

Reply:
Leaderboard information should be updated in the next 24 hours.   Please email slingshot@filecoin.org if you still don't see it after that.